### PR TITLE
Create VeFunds

### PR DESCRIPTION
**Pull request by Emma66886**
# Grant Application Checklist

- [x] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] An Ethereum address for the payment of the milestones is provided inside the application.  
- [x] The total funding amount of the project is up to USD $30k for every grant applications.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

File: applications/VeFunds.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+-Third Milestone(Platform full lunch and maintenance)
+ Duration:1month
+ cost:$5,000
+
+ Total Cost($15,000)
+
+#### Overview
+
+|  | Milestone 1 | Milestone 2 | Total |
```
Comment (libotony) [inline applications/VeFunds.md:73]:
```suggestion
|  | Milestone 1 | Milestone 2 | Milestone 3 | Total| 
```

File: applications/VeFunds.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  UI/UX and more Research:1month
+  Front End And Backend and Mobile App Development :2months 
+  Cost: $9,000
+
+- Second Milestone(Testings and various adjustments)
+- Testing and community feedback : 1month
+  Duration:1month
+  Cost: $1,000
+
+-Third Milestone(Platform full lunch and maintenance)
```
Comment (libotony) [inline applications/VeFunds.md:65]:
```suggestion
- Third Milestone(Platform full lunch and maintenance)
```

File: applications/VeFunds.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that can interact with the deployed smart contracts and backend service. |
+| 0c. | Testing Guide | The code will have proper unit-test coverage (e.g. 90%) to ensure functionality and robustness. In the guide, we will describe how to run these tests |
+| 1 | Smart Contracts |We will develop a smart contract that will enable each user to create campaign and for community to fund each campaign and be able to vote for rlease of funds based on each milestones . We will open source the smart contract and upload the ABI to [B32](https://github.com/vechain/b32). |
+| 2 | Backend | "We will create a backend service that will handle user authentication, manage capigns, and perform several other offchain and onchain functionalities" |
+
+#### Milestone 2  —  Frontend
+| Number | Deliverable | Specification |
+|-|-|-|
+| 1 | User Guide | We will provide a user guide or a demo video to show what features have been done.  |
+...
```
Comment (libotony) [inline applications/VeFunds.md:93]: Please complete the full milestone info, based on your application there are three milestones.

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by Emma66886** [comment]
Thanks @libotony the requested changes has been updated accordingly. Thanks.

File: applications/VeFunds.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+ Total Cost($15,000)
+
+#### Overview
+
+|  | Milestone 1 | Milestone 2 | Milestone 3 | Total |
+| - | - |- | - |
+| Estimated Duration | 90 d | 30 d | 30 d |
+| Full-time equivalent (FTE) | 3 | 1 | 1 |
+| Cost ($ 15,000) | $ 9,000 | $ 1,000 | $ 5,000|
```
Comment (libotony) [inline applications/VeFunds.md:77]: This should be table and the last row should be the total cost.

File: applications/VeFunds.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+|-|-|-|
+| 1 | User Guide | We will provide a user guide or a demo video to show what features have been done.  |
+| 2 | User Testing | The community will be able to test the platform out using the provided guild lines.  |
+
+#### Milestone 3  —  Platform full lunch and maintenance
+| Number | Deliverable | Specification |
+|-|-|-|
+| 1 | Launch and maintenaince | Platform maintenance which include redesigning and addition of new pages and general functionality improvement.  |
+
+...
```
Comment (libotony) [inline applications/VeFunds.md:101]: This line should be removed.

**Comment by Emma66886** [comment]
Changes has been updated. Kindly review it

**Comment by Emma66886** [comment]
Hello @libotony ,How are you doing? 
Just checking back on the changes you requested.

**Comment by libotony** [comment]
> Hello @libotony ,How are you doing? Just checking back on the changes you requested.

@Emma66886 Please take a look at https://github.com/vechain/grant-program/pull/154#discussion_r1236226777, this needs to be updated

**Comment by Emma66886** [comment]
> > Hello @libotony ,How are you doing? Just checking back on the changes you requested.
> 
> @Emma66886 Please take a look at [#154 (comment)](https://github.com/vechain/grant-program/pull/154#discussion_r1236226777), this needs to be updated

Apologies  @libotony I have made change on the request. Thanks

File: applications/VeFunds.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+ Total Cost($15,000)
+
+#### Overview
+
+|  | Milestone 1 | Milestone 2 | Milestone 3 | Total |
+| - | - |- | - | - |
+| Estimated Duration | 90 d | 30 d | 30 d | 150d |
+| Full-time equivalent (FTE) | 3 | 1 | 1 | 5 |
+| Cost (up to $ 15,000)| $ 9,000 | $ 1,000 | $ 5,000 | $ 15,0000 |
```
Comment (libotony) [inline applications/VeFunds.md:77]:
```suggestion
| Cost (up to $ 15,000)| $ 9,000 | $ 1,000 | $ 5,000 | $ 15,000 |
```

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by Emma66886** [comment]
Changes  @libotony  committed

**Comment by Emma66886** [comment]
Hello @libotony just checking back, any response from the team?

**Comment by libotony** [comment]
> Hello @libotony just checking back, any response from the team?

This application is in the review stage, which means it is awaiting review by the committee. We'll get back to you after we collected all votes from the committee.

**Pull request closed without merging**